modules/virt-customizing-storage-profile.adoc

```diff
@@ -19,21 +22,21 @@ If you create a data volume and omit YAML attributes and these attributes are no
 ====
 
 .Procedure
-. Edit the storage profile. In this example, the provisioner is not recognized by the CDI:
+. Edit the storage profile. In this example, the provisioner is not recognized by CDI:
 +
-[source,terminal]
+[source,YAML]
```

Comment:
This one should be `[source,terminal]` as you had it before.

Reply:
Done.